src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle() {
+        this.firstSide = new Random().nextInt(15);
+        this.secondSide = new Random().nextInt(15);
```

Comment:
lets init values outside
exact FigureSupplier should generate figures with random properties:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/ce46f7a7-b894-46cb-8d3f-d171679c653c)
```suggestion
```